### DISCUSSION TITLE
Anyone got a tutorial on how to get llama 75b running through this on runpod?

### DISCUSSION THREAD

**Discussion by austin23cook** [oobabooga/text-generation-webui#208] [General]
Not a lot of interest yet, so I'm having trouble finding instructions.  I could probably figure it out on my own eventually but as I don't even know if 1 A100 80g would be enough I'd rather not start burning money until I'm sure I've got it down.

**Comment by ETA-22** [comment]
This is the guide for this repo: https://github.com/oobabooga/text-generation-webui/wiki/LLaMA-model

doesn’t look too advanced/difficult, if it works.
